packages/manager/apps/pci-savings-plan/src/pages/listing/edit-name/index.tsx

```diff
@@ -1,23 +1,30 @@
 import React, { startTransition, Suspense } from 'react';
 import { useNavigate, useParams, useSearchParams } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { UpdateNameModal } from '@ovh-ux/manager-react-components';
+import {
+  UpdateNameModal,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
 import { useSavingsPlan, useSavingsPlanEditName } from '@/hooks/useSavingsPlan';
 import { REGEX } from '@/utils/savingsPlan';
 
 const EditNameChildren = () => {
+  const { t } = useTranslation('edit-name');
+  const { t: tListing } = useTranslation('listing');
+  const { t: tCreate } = useTranslation('create');
```

Comment:
Better to avoid instantiating multiple instances of `t`: 
```suggestion
  const { t } = useTranslation(['edit-name', 'listing', 'create']);
```